src/queries/duckdb/seattle_buildings_for_map.sql

```diff
@@ -1,16 +1,15 @@
-LOAD spatial;
-LOAD azure;
+LOAD spatial; -- noqa
+LOAD azure;   -- noqa
 
 -- Access the data on Microsoft Azure in this example
 SET azure_storage_connection_string = 'DefaultEndpointsProtocol=https;AccountName=overturemapswestus2;AccountKey=;EndpointSuffix=core.windows.net';
 COPY (
-SELECT
-    names.primary as name,
-    height,
-    level,
-    ST_GeomFromWKB(geometry) as geometry
-FROM read_parquet('azure://release/__OVERTURE_RELEASE/theme=buildings/type=building/*', filename=true, hive_partitioning=1)
-WHERE bbox.xmin > -122.68 AND bbox.xmax < -121.98
-AND bbox.ymin > 47.36 AND bbox.ymax < 47.79
-) TO 'seattle_buildings.geojsonseq'
-WITH (FORMAT GDAL, DRIVER 'GeoJSONSeq', SRS 'EPSG:4326');
+    SELECT
+        names.primary as name,
+        height,
+        level,
+        ST_GeomFromWKB(geometry) as geometry
```

Comment:
All caps for consistency with other changes?

```suggestion
        ST_GEOMFROMWKB(geometry) as geometry
```